tests/unit/amo/components/TestHomepageShelves.js

```diff
@@ -102,26 +108,41 @@ describe(__filename, () => {
   });
 
   it.each([
-    [false, HOMESHELVES_ENDPOINT_COLLECTIONS],
-    [false, HOMESHELVES_ENDPOINT_SEARCH],
-    [true, HOMESHELVES_ENDPOINT_SEARCH_THEMES],
-  ])('passes isTheme as %s when endpoint is %s', (isTheme, endpoint) => {
-    const root = render({
-      shelves: [_createShelf({ endpoint })],
-    });
+    [false, HOMESHELVES_ENDPOINT_COLLECTIONS, null],
+    [false, HOMESHELVES_ENDPOINT_SEARCH, null],
+    [true, HOMESHELVES_ENDPOINT_SEARCH_THEMES, null],
+    [
+      false,
+      HOMESHELVES_ENDPOINT_COLLECTIONS,
+      HOMESHELVES_ADDON_TYPE_EXTENSIONS,
```

Comment:
As mentioned above, just use `ADDON_TYPE_STATIC_THEME` and `ADDON_TYPE_EXTENSION` directly. This goes for all the code in this test suite.